Packs/SplunkPy/Playbooks/playbook-Splunk_Indicator_Hunting.yml

```diff
@@ -319,7 +325,7 @@ tasks:
       append:
         simple: "true"
       key:
-        simple: File.MD5
+        simple: FileMD5
```

Comment:
lets add the unique transformer like we did for Qradar.

Reply:
Done - to all other tasks with the same logic.